riff-raff/app/deployment/Deployments.scala

```diff
@@ -219,7 +219,7 @@ class Deployments(deploymentEngine: DeploymentEngine,
     }
   }
 
-  def await(uuid: UUID): Record = {
-    library()(uuid)()
+  def await(uuid: UUID, atMost: Duration): Record = {
+    Await.result(library.future.flatMap(_(uuid).future), atMost)
```

Comment:
It uses to do exactly this. https://github.com/guardian/riff-raff/commit/128bf1289cfb1288d4b8c7dd630c0690baa9c2fb#diff-ddda48d30e3c7f42e73b8f0cb17d6a76L193 
I wonder why it was changed

Reply:
Looks like Phil (v1) reckoned it would work without. There was also a failed effort to move away from `Agent` so I wonder if that had something to do with it.